core/tx_pool.go

```diff
@@ -635,22 +635,6 @@ func (pool *TxPool) TxPoolPending(enforceTips bool) (map[common.AddressBytes]typ
 	pending := make(map[common.AddressBytes]types.Transactions)
 	for addr, list := range pool.pending {
 		txs := list.Flatten()
-
-		// If the miner requests tip enforcement, cap the lists now
-		if enforceTips && !pool.locals.contains(addr) {
```

Comment:
If you set `enforceTips` parameter to false it would achieve the same functionality. Not a requirement though

Reply:
Changed it, wont push until  get reviews, people might be in the middle of reviews